src/components/Confirm/Confirm.tsx

```diff
@@ -497,6 +497,8 @@ class Confirm extends Component<ConfirmProps, ConfirmState> {
   onRetake = () => {
     const { actions, previousStep } = this.props
 
+    sendEvent('screen_face_selfie_confirmation_retake_button')
```

Comment:
Also this screen is used by almost all flows. We can't add such specific event name like this as it will fire for most flows.